public/script.js

```diff
@@ -7942,44 +7942,45 @@ export function hideSwipeButtons() {
 /**
  * Deletes a swipe from the chat.
  *
- * @param {number?} swipeId - The ID of the swipe to delete. If not provided, the current swipe will be deleted.
+ * @param {number?} [swipeId = null] - The ID of the swipe to delete. If not provided, the current swipe will be deleted.
+ * @param {number?} [mesId = chat.length - 1] - The ID of the message to delete. If not provided, the last message will be deleted.
```

Comment:
Well, we are not deleting the message, still the swipe. But that's very nitpicky now.
Just to be correct...

Reply:
It's just jsdoc. But yeah, if we are working on it, make it correct.